File: cli/vdc/vdc-cli.rb
# ©2021 - BestDodger VDC and server CLI for Ruby
require 'sinatra'
require 'json'
  def createServer(obj)
    obj_json = JSON.parse(obj)
    set :protection, :except => :frame_options
    set :bind, '0.0.0.0'
    set :port, obj_json["port"]

    get '/' do
      if obj_json["type"] == "localText"
        obj_json["localPageText"]
      end
    end
  end
  
  createServer('{"server_name": "NP", "port": 8080, "type": "localText", "localPageText": "<h1>Hello Ruby!</h1>"}')

File: cli/vdc/vdc-cli.js
const vdc = require("cli-vdc-bestdodger")
vdc.loginServer({
  name: "sampleServer",
  development: true,
  api_key: "123456"
})
vdc.startServer({
  portNum: 3000,
  type: "localText",
  localPageText: "Hello Server! This is amazing! just with 11 lines! :) Super good!"
})
